src/applications/gi/utils/render.jsx

```diff
@@ -15,6 +15,7 @@ export const renderSchoolClosingAlert = result => {
       if (currentDate > schoolClosingDate) {
         return (
           <AlertBox
+            className="vads-u-margin-top--0p5"
```

Comment:
@shawkey-daniel @hughes-dustin  This is a quick fix UX specific thing to reduce the margin between the title and the alert